x-pack/plugins/observability_solution/apm/server/routes/data_view/create_static_data_view.ts

```diff
@@ -100,15 +97,6 @@ export async function createStaticDataView({
       dataViewId,
     });
 
-    try {
-      await dataViewService.delete(DO_NOT_USE_LEGACY_APM_STATIC_DATA_VIEW_ID);
```

Comment:
I'm just thinking that we might want to backport this change to 8.12 and 8.13 to fix bugs like https://github.com/elastic/sdh-apm/issues/1272 (internal). wdyt?

Reply:
I've already added the `8.13.1` label, it won't make it on time for `8.13.0`. And there's no planed release for `8.12`